README-DEVELOPER.md

```diff
@@ -22,6 +22,7 @@ Manual installation takes place in two steps. The code must be installed via `np
 To install the dependencies:
 ``` sh
 npm install
+npm run build:wasm
```

Comment:
I expect this will need to be explicitly run on some additional CI workflows?

Reply:
I think this is already here: https://github.com/yoctoproject/vscode-bitbake/blob/98c125d843a4c0156ae372203751e6ef03a26451/.github/workflows/main.yml#L66

Is that what you mean?

Reply:
Yes!